hw/ip/entropy_src/doc/dv_plan/index.md

```diff
@@ -94,5 +91,4 @@ $ make TEST_NAME=entropy_src_sanity
 ```
 
 ## Testplan
-<!-- TODO: uncomment the line below after adding the testplan
-{{</* testplan "hw/ip/entropy_src/data/entropy_src_testplan.hjson" */>}} -->
+{{< testplan "hw/ip/entropy_src/data/entropy_src_testplan.hjson" >}} -->
```

Comment:
Stray `-->` at the end.

Reply:
@senelson7 can you remove `-->`, then we can merge this PR

Reply:
Done